webpack.config.js

```diff
@@ -11,6 +12,10 @@ const baseConfig = {
   module: {
     loaders: [
       {
+        test: /\.css$/,
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'),
```

Comment:
I believe that's the ideas, and that the deployment would be triggered by CircleCi